Core/include/Acts/Vertexing/AdaptiveMultiVertexFitter.hpp

```diff
@@ -204,6 +213,18 @@ class AdaptiveMultiVertexFitter {
       const Linearizer_t& linearizer,
       const VertexingOptions<InputTrack_t>& vertexingOptions) const;
 
+  /// @brief Performs a simultaneous fit of all vertices in
+  /// state.vertexCollection
+  ///
+  /// @param state Fitter state
+  /// @param linearizer Track linearizer
+  /// @param vertexingOptions Vertexing options
+  ///
+  /// @return Result<void> object
+  Result<void> fit(
+      State& state, const Linearizer_t& linearizer,
+      const VertexingOptions<InputTrack_t>& vertexingOptions) const;
```

Comment:
This just moves without changes?

Reply:
I deleted the function `fit` and renamed `fitImpl` -> `fit`